app/src/main/AndroidManifest.xml

```diff
@@ -8,6 +8,7 @@
     <uses-permission android:name="android.permission.READ_SYNC_STATS" />
     <uses-permission android:name="android.permission.WRITE_SYNC_SETTINGS" />
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
+    <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
```

Comment:
Should not change this file

Reply:
Actually these were changes done previously on master branch of apps-android-commons

Reply:
Github only highlights the changes between your branch and master.

See https://github.com/commons-app/apps-android-commons/blob/78ca1fe64eff484723aa815a82b60bc33483ed3e/app/src/main/AndroidManifest.xml